### PR TITLE
Update configure-docker.md

### PR DESCRIPTION
**Pull request by arthuo**
Overwrite with new ones_zh

### CODE REVIEW

File: site/zh-CN/adminGuide/configure-docker.md [hunk 1]
```diff
@@ -7,36 +7,36 @@ order: 0
...
 
-Check the following links for more information about each parameter.
+有关每个参数的更多信息，请查看以下链接。
 
-Sorted by:
+排序：
 
 <div class="filter">
-<a href="#component">Components or dependencies</a> <a href="#purpose">Configuration purposes</a> 
+<a href="#component">组件或依赖项</a> <a href="#purpose">配置目的</a>
```
Comment (LocoRichard) [inline site/zh-CN/adminGuide/configure-docker.md:39]: `依赖` is OK

File: site/zh-CN/adminGuide/configure-docker.md [hunk 2]
```diff
@@ -45,8 +45,8 @@ Sorted by:
 <table id="component">
 <thead>
   <tr>
-    <th>Dependencies</th>
-    <th>Components</th>
+    <th>依赖项</th>
```
Comment (LocoRichard) [inline site/zh-CN/adminGuide/configure-docker.md:48]: `依赖` is OK

File: site/zh-CN/adminGuide/configure-docker.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ Sorted by:
     </td>
   </tr>
   <tr>
-    <td>Data and meta</td>
+    <td>数据和元</td>
```
Comment (LocoRichard) [inline site/zh-CN/adminGuide/configure-docker.md:111]: 数据与元数据

File: site/zh-CN/adminGuide/configure-docker.md [hunk 4]
```diff
@@ -120,7 +120,7 @@ Sorted by:
     </td>
   </tr>
   <tr>
-    <td>Administration</td>
+    <td>管理</td>
```
Comment (LocoRichard) [inline site/zh-CN/adminGuide/configure-docker.md:123]: 集群管理

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Greetings @arthuo , Thank you for your kind contribution. I've made a few comments.
It has been a tough spring due to the pandemic. I hope that you have taken good care of you and your family.

**Comment by arthuo** [comment]
> Greetings @arthuo , Thank you for your kind contribution. I've made a few comments. It has been a tough spring due to the pandemic. I hope that you have taken good care of you and your family.

Thanks. You should also take good care of yourself.

**Pull request merged**